css/about.css

```diff
@@ -0,0 +1,167 @@
+/* I tried my best but couldn't go far enough to replicate pixel-for-pixel design */
+
+
+/* My code starts here */
+
+.navigation {
```

Comment:
Try playing around with the "align-items" property here to get your links and image centered with each other

Reply:
Fixed thanks.